### PR TITLE
Release v0.16.0

### PR DESCRIPTION
**Pull request by zramsay**
- includes what we need for the platform 0.16.0 release
- sort of breaks git flow but ...
- this repo will either be deprecated and/or moved into the burrow repo; let's leave master as the only branch ... we can version bump a patch for any critical fix

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
-# Install eris-keys, a go app for development signing
-ENV TARGET eris-keys
+# Install monax-keys, a go app for development signing
+ENV TARGET monax-keys
 ENV REPO $GOPATH/src/github.com/monax/keys
 
+# required for testing; should be removed
+RUN apk --no-cache --update add openssl
+
```
Comment (NodeGuy) [inline Dockerfile]: Should this be removed?

File: Dockerfile [hunk 2]
```diff
@@ -16,8 +19,7 @@ RUN cd $REPO/cmd/$TARGET && \
 
 # build customizations start here
 # install mint-key [to be deprecated]
-# NOTE: repo needs to refer eris-ltd to ensure the gopath to have old
-# directory name for mint-client; ready for deprecation
+# NOTE: repo must remain [eris-ltd] to Godep reasons
 ENV MONAX_KEYS_MINT_REPO github.com/eris-ltd/mint-client
```
Comment (NodeGuy) [inline Dockerfile]: Are you sure it can't be renamed to `monax`?

Comment (zramsay) [inline Dockerfile]: nope. tried and Godeps cried.

File: Dockerfile (hunk 1 above)
Comment (zramsay) [inline Dockerfile]: - the comment should say `should be removed eventually`.
- it's required by the CI (see https://github.com/monax/keys/blob/develop/test.sh#L66 and https://github.com/monax/keys/blob/develop/circle.yml#L23)
- i suspect it was removed from the base alpine image or somewhere else along the image pipeline & decided it wasn't work the effort refactoring the tests with a new paradigm

**Pull request merged**